modules/dop/services/autotest_v2/scene.go

```diff
@@ -525,6 +525,32 @@ func (svc *Service) ExecuteDiceAutotestSceneStep(req apistructs.AutotestExecuteS
 		apiTestStr = strings.ReplaceAll(apiTestStr, expression.OldLeftPlaceholder+expression.Params+"."+param.Name+expression.OldRightPlaceholder, expression.ReplaceRandomParams(param.Temp))
 	}
 
+	var pipelinePageListRequest = apistructs.PipelinePageListRequest{
+		PageNum:  1,
+		PageSize: 10,
```

Comment:
why get 10 but only use one (index 0)?

Reply:
Really shouldn't query 10

Get the subscript 0, just to get the latest end record，I added sorting by id should be no problem